README.md

```diff
@@ -159,7 +159,7 @@ SIG-specific GitHub discussions.
 |------|--------------|---------------|---------------|-----------------|--------------------------------|
 | <a id="sig-android"></a>Android: SDK + Automatic Instrumentation | Tuesday at 08:00 PT | [Google Doc](https://docs.google.com/document/d/1W72oElAJuYx3efM9wkH5l1iksdfwmTSkVl-H7KHTLnY) | [#otel-android](https://cloud-native.slack.com/archives/C05J0T9K27Q) | [calendar-android](https://groups.google.com/a/opentelemetry.io/g/calendar-android) | [Trask Stalnaker](https://github.com/trask) |
 | <a id="sig-collector"></a>Collector | Alternating between Tuesday at 17:00 PT, Wednesday at 09:00 PT, and Wednesday at 05:00 PT | [Google Doc](https://docs.google.com/document/d/1r2JC5MB7GupCE7N32EwGEXs9V_YIsPgoFiLP4VWVMkE) | [#otel-collector](https://cloud-native.slack.com/archives/C01N6P7KR6W) | [calendar-collector](https://groups.google.com/a/opentelemetry.io/g/calendar-collector) | [Pablo Baeyens](https://github.com/mx-psi) |
-| <a id="sig-cpp"></a>C++: SDK | Alternating between Monday at 13:00 PT and Wednesday at 09:00 PT PT | [Google Doc](https://docs.google.com/document/d/1i1E4-_y4uJ083lCutKGDhkpi3n4_e774SBLi9hPLocw) | [#otel-cpp](https://cloud-native.slack.com/archives/C01N3AT62SJ) | [calendar-cpp](https://groups.google.com/a/opentelemetry.io/g/calendar-cpp) | [Severin Neumann](https://github.com/svrnm) |
+| <a id="sig-cpp"></a>C++: SDK | Alternating between Monday at 13:00 PT and Wednesday at 09:00 PT | [Google Doc](https://docs.google.com/document/d/1i1E4-_y4uJ083lCutKGDhkpi3n4_e774SBLi9hPLocw) | [#otel-cpp](https://cloud-native.slack.com/archives/C01N3AT62SJ) | [calendar-cpp](https://groups.google.com/a/opentelemetry.io/g/calendar-cpp) | [Severin Neumann](https://github.com/svrnm) |
```

Comment:
thanks for catching that!
can you apply the same change in the `sigs.yml` and run the script to update the README? This table is auto generated from that file